packages/blocks/src/components/question-wrapper/index.js

```diff
@@ -27,7 +27,7 @@ const StyledQuestionWrapper = styled.div`
 	}
 
 	&.is-error {
-		border-color: var( --color-error );
+		border: 1px solid var( --color-error ) !important;
```

Comment:
Somewhat of a nit... by fixing the border size to `1px` this can feel a little inconsistent depending on what the actual border size was.  
I wonder if it might be a better idea to use `box-shadow` for the error outline - maybe even put it on a ::before or ::after so it doesn't affect the shadow for question wrappers. Adding a pseudo element potentially also saves us the `!important` but it's not a huge deal here I think. What do you think?

Reply:
I think the pseudo-element is worth a try. Let me see what I can get.

Reply:
I think we will be stuck to use `!important` here, hence, rolling back the change (use only `border-color`) and just append the `!important` directive.

One issue that comes to mind is if the user set the border color to some very similar `--color-error` color.

Reply:
We could adjust it to use `!important` only on `border-color` and add a default `border-width` but, this would allow the user to set the `border-width` to 0 and, this way, they wouldn't see the border for the error state.